tests/test_scoring.py

```diff
@@ -75,26 +86,103 @@ def test_create_list_events():
+    npt.assert_equal(report["Clean MEMA duration"], len(classif))
+    npt.assert_equal(report["Total MEMA burst duration"], 6)
+    npt.assert_equal(report["Number of MEMA bursts per hour"], 3600/len(classif)*2)
+    npt.assert_equal(report["Total number of MEMA episodes"], 1)
+    npt.assert_equal(report["Number of MEMA bursts per episode"], 2.0)
+    npt.assert_equal(report["Number of MEMA episodes per hour"], 3600/len(classif)*1)
+    npt.assert_equal(report["Total number of MEMA burst"], 2)
+    npt.assert_equal(report["Mean duration of MEMA episode"], len(classif))
 
 
+    classif.extend([False, False, False, False, False, False, True, False, False, True, False, False, True])
+    report = generate_MEMA_report(classif, 1, delim=3)
+    npt.assert_equal(len(report), 8)
+    npt.assert_equal(report["Clean MEMA duration"], len(classif))
+    npt.assert_equal(report["Total MEMA burst duration"], 9)
+    npt.assert_equal(report["Number of MEMA bursts per hour"], 3600/len(classif)*5)
+    npt.assert_equal(report["Total number of MEMA episodes"], 2)
+    npt.assert_equal(report["Number of MEMA bursts per episode"], 2.5)
+    npt.assert_equal(report["Number of MEMA episodes per hour"], 3600/len(classif)*2)
+    npt.assert_equal(report["Total number of MEMA burst"], 5)
+    npt.assert_equal(report["Mean duration of MEMA episode"], 7.5)
+
+
+def test_generate_bruxism_report2():
+    classif = [False, False, False, True, True, True]
+    report = generate_bruxism_report(classif, 1, 3)
+    npt.assert_equal(len(report), 13)
+
+    npt.assert_equal(report["Clean data duration"], len(classif))
+    npt.assert_equal(report["Total burst duration"], 3)
+    npt.assert_equal(report["Total number of burst"], 1)
+    npt.assert_almost_equal(report["Number of bursts per hour"], 3600/len(classif)*1)
+    npt.assert_equal(report["Total number of episodes"], 1)
+    npt.assert_equal(report["Number of bursts per episode"], 1)
+    npt.assert_almost_equal(report["Number of episodes per hour"], 3600/len(classif)*1)
+    npt.assert_almost_equal(report["Number of tonic episodes per hour"], 3600/len(classif)*1)
+    npt.assert_almost_equal(report["Number of mixed episodes per hour"], 3600/len(classif)*0)
+    npt.assert_almost_equal(report["Number of phasic episodes per hour"], 3600/len(classif)*0)
+    npt.assert_almost_equal(report["Mean duration of mixed episode"], np.nan)
+    npt.assert_almost_equal(report["Mean duration of phasic episode"], np.nan)
+    npt.assert_almost_equal(report["Mean duration of tonic episode"], 3)
+
+def test_generate_report_fails_noparams():
+    """check if the parameters are required or test fails"""
+    classif = [False, False]
+    with pytest.raises(TypeError, match=f"missing 2 required positional arguments: 'time_interval' and 'delim'"):
+        generate_bruxism_report(classif)
+    with pytest.raises(TypeError, match=f"missing 2 required positional arguments: 'time_interval' and 'delim'"):
+        generate_MEMA_report(classif)
```

Comment:
this part will heavily extended because there was no actual test on crucial values

Reply:
Seems a little overkill but why not!

Reply:
I know, but image one line was wrong. Or worse, imagine that we put a mistake in the future.

We should always test all outputs.